about.html

```diff
@@ -22,24 +21,19 @@
 <h3>Applications</h3>
 
 <ul>
-<li>PcManFm-qt - File Manager</li>
-<li>Lximage-qt - Image Viewer</li>
-<li>QTerminal - Terminal Emulator</li>
-<li>Qps - Process Viewer</li>
-<li>Screengrab - Screen Capturer</li>
-<li>LXQt-archiver - Archive Manager</li>
-<li>LXQt-runner - Application launcher and calculator</li>
+<li>lxqt-runner — Launch applications and do calculations</li>
+<li>PCManFM-Qt — Browse files</li>
+<li>LXImage-Qt — View images</li>
+<li>QTerminal — Use the terminal</li>
+<li>ScreenGrab — Capture screenshots</li>
+<li>LXQt Archiver — Manage archives</li>
+<li>Qps — Manage processes</li>
 </ul>
 <br/>
 
-<p>LXQt is available for all major Linux and BSD distributions so you may just try it
-out on your system or in a VM.<br>
```

Comment:
Why introduce the concept "VM" for curious people clicking "About"?

Reply:
I see many users that hesitate  to install another DE along their existing one. 
Do you want to try out other DE's by installing them immediately on HD?

Reply:
Yes. That however is not the issue.
Notice the KDE/Plasma approach, installing a whole new distro.